### ISSUE TITLE
Errors seem to be output twice

### ISSUE THREAD

**Issue by jeffijoe** [MarkHerhold/palin#4]
``` js
logger.error('Hah!', new Error())
```

This outputs the error + it's stack twice.

**Comment by MarkHerhold** [comment]
Intesting. I'll see if I can reproduce what you are seeing tonight. What
version of Bristol and node.js are you on?

**Comment by jeffijoe** [comment]
Node 6.2.2, latest Bristol and Palin. :)

**Comment by MarkHerhold** [comment]
This is a really interesting issue.

**node.js v4, v5.5 produces**:

```
  20:52:16:198 ERROR Hah! (/Users/Mark/projects/palin-issue/index.js:5)
    →  [Error]
    →  Error
    →      at Object.<anonymous> (/Users/Mark/projects/palin-issue/index.js:5:22)
    →      at Module._compile (module.js:413:34)
    →      at Object.Module._extensions..js (module.js:422:10)
    →      at Module.load (module.js:357:32)
    →      at Function.Module._load (module.js:314:12)
    →      at Function.Module.runMain (module.js:447:10)
    →      at startup (node.js:148:18)
    →      at node.js:405:3
```

**node.js v6 produces**:

```
  20:52:25:154 ERROR Hah! (/Users/Mark/projects/palin-issue/index.js:5)
    →  Error
    →      at Object.<anonymous> (/Users/Mark/projects/palin-issue/index.js:5:22)
    →      at Module._compile (module.js:541:32)
    →      at Object.Module._extensions..js (module.js:550:10)
    →      at Module.load (module.js:458:32)
    →      at tryModuleLoad (module.js:417:12)
    →      at Function.Module._load (module.js:409:3)
    →      at Function.Module.runMain (module.js:575:10)
    →      at startup (node.js:160:18)
    →      at node.js:456:3
    →  Error
    →      at Object.<anonymous> (/Users/Mark/projects/palin-issue/index.js:5:22)
    →      at Module._compile (module.js:541:32)
    →      at Object.Module._extensions..js (module.js:550:10)
    →      at Module.load (module.js:458:32)
    →      at tryModuleLoad (module.js:417:12)
    →      at Function.Module._load (module.js:409:3)
    →      at Function.Module.runMain (module.js:575:10)
    →      at startup (node.js:160:18)
    →      at node.js:456:3
```

This is a good find @jeffijoe :+1:

**Comment by MarkHerhold** [comment]
Landed and published in palin@2.0.0
